playwright-e2e/tests/dsm/playwright.config.ts

```diff
@@ -1,5 +1,9 @@
 import type { PlaywrightTestConfig } from '@playwright/test';
 import testConfig from 'playwright.config';
+import * as dotenv from 'dotenv';
+import path from 'path';
```

Comment:
I think it's not needed for **CircleCi**
@aweng98

Reply:
As it is already done in the main config file:

![Screenshot 2022-12-15 at 18 03 07](https://user-images.githubusercontent.com/77500504/207881132-7deae3c5-3163-4763-b79d-eaa413f8daba.png)

About .env, these commands are taking care of:

![Screenshot 2022-12-15 at 18 04 18](https://user-images.githubusercontent.com/77500504/207881191-cd3cac06-c7de-4369-ae06-7b9fa3e0d4b0.png)

Reply:
right, not needed.

Reply:
If I remove this and run `npm run test:e2e:dsm` then I get `Error: Invalid parameter: DSM user email is undefined or null.` I made sure to fill in the values for DSM_USER_EMAIL and DSM_USER_PASSWORD so they are not null. I am probably doing something wrong with the config, but I cannot figure out what.